src/viewer.js

```diff
@@ -1252,6 +1252,15 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
     },
 
 
+    //
+    /**
+     * @function
+     * @return {Boolean} returns true if the viewer is in fullscreen
+     */
+     isFullScreen: function () {
+        return $.setFullScreen.fullScreen;
```

Comment:
As discussed in https://github.com/openseadragon/openseadragon/issues/1320, we want this to tell whether the viewer in question is fullscreen (because there could be more than one viewer). You can do that by testing `$.isFullScreen() && this.isFullPage()`.